### PR TITLE
Normalization package for registry events

### PR DESCRIPTION
**Pull request by oshezaf**
- Parser for MDE, Sysmon and WIndows
  - One hunting query

### CODE REVIEW

**Comment by oshezaf** [comment]
@ashwin-patil , @shainw : just one hunting query here to review.

File: Parsers/ASimRegistry/RegistryEventEmpty.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Normalization:
+  Schema: RegistryEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Registry Schema
+  Link: https://aka.ms/AzSentinelRegistryEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AzSentinelNormalization
+Description: |
+  The purpose of this KQL function is to generate and guarentee the schema columns for the ASIM Registry Event schema
```
Comment (shainw) [inline Parsers/ASimRegistry/RegistryEventEmpty.yaml:16]: misspelled - guarantee - but maybe we should say this in a different way such as "make available the default schema" or something like that.

**Review by shainw — CHANGES_REQUESTED** [review]

File: Parsers/ASimRegistry/RegistryEventEmpty.yaml (hunk 1 above)
Comment (oshezaf) [reply Parsers/ASimRegistry/RegistryEventEmpty.yaml:16]: Changed per your suggestion

File: Parsers/ASimRegistry/ARM/RegistryEventGeneric/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Source Agnostic ASIM RegistryEvent Normalization Parser
+
+This template deploys the Source Agnostic ASIM RegistryEvent schema parser. The parser is a part of the Azure Sentinel Information Model.
```
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimRegistry/ARM/RegistryEventGeneric/README.md:3]: Fixed

**Comment by oshezaf** [comment]
All requested changes resolved

**Pull request merged**